spec/requests/articles_spec.rb

```diff
@@ -1,11 +1,78 @@
 require 'rails_helper'
 
 RSpec.describe "Articles", type: :request do
+
+  before(:all) do
```

Comment:
I would check the difference between `before(:each)` and `before(:all)`.

Reply:
If you look at the tests I wrote for https://github.com/jensen/conveyer-coop you can see that the convention is to use `let` for these types of definitions.